testing.rst

```diff
@@ -619,15 +619,58 @@ This allows you to create all types of requests you can think of:
     :ref:`framework.test <reference-framework-test>` option is enabled).
     This means you can override the service entirely if you need to.
 
-.. caution::
+Multiple requests in one test
+.............................
+
+After you send one request, subsequent ones will make the client reboot
+the kernel, recreating the container from scratch.
+This ensures that requests are "isolated" using "new" service objects,
+but can cause some unexpected behaviors. For example, the security token will
```

Comment:
```suggestion
This ensures that requests are "isolated" using "new" service objects.
However, this can cause some unexpected behaviors. For example, the security token will
```